sdk/storage/inc/datalake/datalake.hpp

```diff
@@ -0,0 +1,9 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#pragma once
+
+#include "datalake/datalake_utilities.hpp"
```

Comment:
This header shouldn't be directly include by `sdk/storage/inc/datalake/datalake.hpp`

Reply:
Yeah, if all things within utilities are in `Details` namespace, I agree with you.